web/src/app/schedules/ScheduleOverrideForm.tsx

```diff
@@ -30,7 +30,27 @@ const query = gql`
   }
 `
 
-export default function ScheduleOverrideForm(props) {
+interface ScheduleOverrideFormValue {
+  addUserID: string
+  removeUserID: string
+  start: string
+  end: string
+}
+
+interface ScheduleOverrideFormProps {
+  scheduleID: string
+  value: ScheduleOverrideFormValue
+  add?: boolean
+  remove?: boolean
+  disabled: boolean
+  errors: Array<FieldError>
+  onChange?: (value: ScheduleOverrideFormValue) => void
```

Comment:
Looks like `onChange` was required before the conversion. 
```suggestion
  onChange: (value: ScheduleOverrideFormValue) => void
```